src/common/components/select-repository-search/index.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component, PropTypes } from 'react';
+import styles from './search.css';
+
+export default class SelectRepositorySearch extends Component {
+
+  render() {
+    let id = this.props.id || 'search-repositories';
```

Comment:
I wouldn't set id default value as it will be easily forgotten and we can end up with many inputs with same id.

I suppose if input is inside of the label id is really optional, so maybe we could have it without any id unless dev specifies it?